common/src/commonMain/kotlin/entity/Interactions.kt

```diff
@@ -156,19 +156,19 @@ public sealed class Choice {
 
     public data class IntegerChoice(
         override val name: String,
-        override val nameLocalizations: Optional<Map<Locale, String>?>,
+        override val nameLocalizations: Optional<Map<Locale, String>?> = Optional.Missing(),
```

Comment:
i think it should be fine to not have a default if it's not a class marked with `@Serializable`. that's what the KSP processor did too.

Reply:
These classes should be serializable as `Choice` is serializable?

Reply:
they are, but the default for optional is only needed with generated serializers (that will only be generated if the annotation is present), in this case there exists a [manual serializer implementation](https://github.com/kordlib/kord/blob/569b113a6ae0dc6c352be5cd2491634af65e16a1/common/src/commonMain/kotlin/entity/Interactions.kt#L175) that handles the optionals correctly